src/archemist/core/optimisation/recipe_converter.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import yaml
+from archemist.core.persistence.yaml_handler import YamlHandler
+from nested_lookup import nested_update
+from pathlib import Path
+import logging
+import re
+
+class RecipeUpdate:
```

Comment:
Change class name to RecipeGenerator and the filename respectively to recipe_generator.py

Reply:
Done

Reply:
Changed